### PR TITLE
Add `gasPerConsSec` statistic

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
- In the first transaction handled per (consensus) second, updates a new `gasPerConsSec` running average with the total gas consumed during the last (consensus) second.

### CODE REVIEW

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM!

**Review by anighanta — APPROVED** [review]

File: hedera-node/src/main/java/com/hedera/services/state/logic/NetworkCtxManager.java [hunk 1]
```diff
@@ -193,10 +220,23 @@ public static boolean inSameUtcDay(Instant now, Instant then) {
...
+	void setConsensusSecondJustChanged(boolean consensusSecondJustChanged) {
+		this.consensusSecondJustChanged = consensusSecondJustChanged;
+	}
+
+	long getGasUsedThisConsSec() {
+		return gasUsedThisConsSec;
+	}
+
+	void setGasUsedThisConsSec(long gasUsedThisConsSec) {
+		this.gasUsedThisConsSec = gasUsedThisConsSec;
```
Comment (qnswirlds) [inline hedera-node/src/main/java/com/hedera/services/state/logic/NetworkCtxManager.java:237]: nit: add `final`

File: hedera-node/src/test/java/com/hedera/services/state/logic/NetworkCtxManagerTest.java [hunk 2]
```diff
@@ -66,10 +62,11 @@
...
 class NetworkCtxManagerTest {
-	private int issResetPeriod = 5;
-	private Instant sometime = Instant.ofEpochSecond(1_234_567L);
-	private Instant sometimeSameDay = sometime.plusSeconds(issResetPeriod + 1L);
-	private Instant sometimeNextDay = sometime.plusSeconds(86_400L);
+	private final int issResetPeriod = 5;
+	private final long someGasUsage = 8910L;
+	private final Instant sometime = Instant.ofEpochSecond(1_234_567L);
+	private final Instant sometimeSameDay = sometime.plusSeconds(issResetPeriod + 1L);
+	private final Instant sometimeNextDay = sometime.plusSeconds(86_400L);
```
Comment (qnswirlds) [inline hedera-node/src/test/java/com/hedera/services/state/logic/NetworkCtxManagerTest.java:69]: nit: those could be `static` as well.

**Review by qnswirlds — APPROVED** [review]
Just some nits. Overall LGTM.

**Pull request merged**